op3_ball_detector/include/op3_ball_detector/ball_detector_config.h

```diff
@@ -123,7 +127,7 @@ class BallColorConfig
   int sampleLightVal();
   int getMedianRVal(int x_val);
   void updateDistribution(std::vector<double> light_range, std::vector<double> light_weights);
-  void adjustWeightsWithLightVal(int light_val, int adjust_val, std::vector<double> &light_weights);
+  void adjustWeightsWithLightVal(int light_val, bool reward, std::vector<double> &light_weights);
```

Comment:
I think something like bool is_reward is a bit more descriptive of the variable

Reply:
Changed in ea0f08d